quantify-basecounter.wdl

```diff
@@ -0,0 +1,24 @@
+import "tasks/biopet.wdl" as biopet
+
+workflow QuantifyBaseCounter {
```

Comment:
Maybe a subworkflow for a single tool is a bit to much

Reply:
Indeed, I suppose (also in regards to the general comment you made) we could put them all in one workflow.